src/Symfony/Bundle/WebServerBundle/WebServer.php

```diff
@@ -145,12 +144,9 @@ public function isRunning($pidFile = null)
      */
     private function createServerProcess(WebServerConfig $config)
     {
-        $finder = new PhpExecutableFinder();
-        if (false === $binary = $finder->find()) {
-            throw new \RuntimeException('Unable to find the PHP binary.');
-        }
+        $executable = $config->getExecutable();
 
-        $process = new Process(array($binary, '-S', $config->getAddress(), $config->getRouter()));
+        $process = new Process(array($executable, '-S', $config->getAddress(), $config->getRouter()));
```

Comment:
I believe we should "explode" `$executable` string. Depending on how the Process class is escaping the array, the first one will be the command name, see the example:

```bash
$ 'echo -e test'
bash: echo -e test: command not found
```

If you pass a whole string as the command, it will be interpreted as its name. I did not test it, but I believe it will do the same in the end.